src/measurement_kit/cmd/telegram.cpp

```diff
@@ -0,0 +1,33 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include "../cmdline.hpp"
+
+namespace telegram {
+
+#define USAGE                                       \
+    "usage: measurement_kit [options] telegram\n"   \
+
+int main(std::list<Callback<BaseTest &>> &initializers, int argc, char **argv) {
+    mk::nettests::TelegramTest test;
+    int ch;
+    while ((ch = getopt(argc, argv, "B:f:")) != -1) {
```

Comment:
Here you should clear the options string, if no options are recognized.

Reply:
Whoops, done.